### PR TITLE
Add user guide about slora serving 

### PR DESCRIPTION
**Pull request by AllentDan**

### CODE REVIEW

**Review by grimoire — APPROVED** [review]
LGTM

File: docs/zh_cn/llm/api_server_lora.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+# Server S-LoRA
```
Comment (lvhan028) [inline docs/zh_cn/llm/api_server_lora.md:1]: LoRA 推理服务

File: docs/en/llm/api_server_lora.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
+# Server S-LoRA
```
Comment (lvhan028) [inline docs/en/llm/api_server_lora.md:1]: Serving LoRA

**Review by lvhan028 — APPROVED** [review]

**Comment by lvhan028** [comment]
The linting issue is being addressed in https://github.com/InternLM/lmdeploy/pull/2038
Will merge this PR after #2038

**Pull request merged**